### DISCUSSION TITLE
upgrading from nbright 3.5.7 assistance

### DISCUSSION THREAD

**Discussion by Puppyknucks** [openstore-ecommerce/OpenStore#160] [Q&A]
is there a process document of upgrading from 3.5.7 to the newest open door?  I installed 8.6.3 and nbrightbuy 4.1.5 but that messed up the 3.5.7 store I had setup and now I'm restoring the backups on the sites so i can try again fresh.  
I've tried installing on the host and the individual sites, I've tried creating new pages and a new back office page with nbrightstoreBO and I keep getting connection reset errors.
It looks like your forum is off line.

Thanks for your time
Jeff Knox
www.puppyknucks.com

**Comment by DNNMonster** [comment]
I used the simple upgrade txt file you can find here: https://github.com/openstore-ecommerce/OpenStore/blob/master/Installation/Readme_NBS_Upgrade.txt

I successfully performed the upgrade but it was too long ago for me to remember if there were any gotcha's. I don't think there were.

While your doing your upgrade, if you see room for improvements to the txt file that would be helpful for others it would be great to have you contribute those.

**Comment by Puppyknucks** [comment]
I’ll give it a try!  How do you like openstore, is it working well for you?

**Comment by Puppyknucks** [comment]
I just looked thru this text doc and I’m up for trying method 1 but I still can’t find any process docs for ‘creating a store’.  I think the setup is really easy but its been years since I did nbright and can’t recall the process.

**Comment by DNNMonster** [comment]
There is some pretty good store set up documentation for you.  It's the place to start.  

https://doc.openstore-ecommerce.com/

**Comment by Puppyknucks** [comment]
I thought I remembered this software building the entire store and all pages.  Either way, it did not, so I am manually rebuilding my store on all new pages to keep it separate from the Nbright store I currently have installed.

 

Here are a few changes that might help the integrator guild and where my install came to a crashing halt

 

https://doc.openstore-ecommerce.com/

Integrator Guide


Pages and modules


Add “Create a new store page and create the following child pages. None of them need to be viewable in the menu after you are done with building your store.”

 

Hooks:

2. catalog page

The image shows “product List” (OS_ProductDisplayAjax)

The modules installed are Item List (OS_ItemListRazor) and Product View (OS_ProductDisplayAjax)
Item List is the shopping list you put inside the client account
Product view is the product catalogue you add to your store page

 

3. Detail page
here is where I had to stop because there is no OS_ProductDisplay module to install.  So I can’t add a detail view to any products

**Comment by leedavi** [comment]
I cannot see "OS_ProductDisplay" module on the documentation? 

https://doc.openstore-ecommerce.com/ 

I see the template.  But, in any case the module "OS_ProductDisplay" was rename to "OS_ProductView" in v4.0.4


        v04.00.04.00
        - Add localized hidden product.
        - Allow DDRMENU token to select parentitemid. Token can now be [CAT] or [CAT:{xxxxx}]. {xxxxx} = categoryref.
        - Improve Product Admin JS
        - Add Multi-checkbox select for Categories and Properties, in the Product Admin.
        - Replace OS_ProductDisplay and OS_ProductDisplayAjax with OS_ProductView, to remove duplicate code.
        - Improved Scheduler error.
        - Rename OS_ProductAjaxFilter to OS_ProductFilter.
        - WARNING IF UPGRADING: Read "Readme_Upgrade_to_4.0.4.txt"
